types/search_options.go

```diff
@@ -0,0 +1,37 @@
+package types
+
+type SearchOptions struct {
+	Type   string
+	From   int
```

Comment:
`From` and `Size` must be declared as `int*` as we need a way to tell whether the option has been set or not